app/tests/integration/test_filter_expression_validation.py

```diff
@@ -1,16 +1,12 @@
-def test_filter_expressions_with_matching_firefox_versions(
-    base_url, selenium
-):
+def test_filter_expressions_with_matching_firefox_versions(base_url, selenium):
     selenium.get("about:blank")
     with open("utils/filter_expression.js") as js:
         with selenium.context(selenium.CONTEXT_CHROME):
             script = selenium.execute_script(js.read(), 80.1, 80)
     assert script is True
 
 
-def test_filter_expressions_with_mismatching_firefox_versions(
-    base_url, selenium
-):
+def test_filter_expressions_with_mismatching_firefox_versions(base_url, selenium):
```

Comment:
Is this just linting? Surprised it didn't get caught sooner.

Reply:
Yeah I think what's happening is black is set to format that path but not check that path otherwise it shoudln't have landed in main.  I'm gonna fix that now.